src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -14,27 +14,145 @@
 limitations under the License.
 -->
 
-<div layout="column" layout-margin layout-align="center center">
-  Displaying details for replica set {{ctrl.replicaSetDetail.name}} in namespace
-  {{ctrl.replicaSetDetail.namespace}}.
-  <div>
-    <table>
-      <thead>
-        <tr>
-          <th>Pod</th>
-          <th>Start time</th>
-          <th>IP</th>
-          <th>Node</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr ng-repeat="pod in ctrl.replicaSetDetail.pods">
-          <td>{{pod.name}}</td>
-          <td>{{pod.startTime}}</td>
-          <td>{{pod.podIP}}</td>
-          <td>{{pod.nodeName}}</td>
-        </tr>
-      </tbody>
-    </table>
+<md-content layout="row" layout-fill flex>
+  <div class="kd-replicasetdetail-sidebar" layout="column">
+    <div flex layout-align="start center" layout="row" class="kd-header-option-panel">
+      <a flex-offset="10" ui-sref="replicasets">
+        <i class="material-icons kd-align-mid kd-font-lg">keyboard_backspace</i>
+      </a>
+      <span flex-offset="5" flex class="kd-align-mid md-title">
+        {{ctrl.replicaSetDetail.name}}
+      </span>
+    </div>
+    <div flex-offset="5" flex layout-align="start start" layout-padding layout="row">
+      <div flex layout="column">
+        <h3 layout="row" class="kd-primary md-body-2">
+          <div flex="55">
+            <i class="kd-align-mid material-icons">refresh</i>
```

Comment:
Hmm... If the official spec says so. Let's keep that in mind and add to a
style guide when we have one.


On Fri, Dec 4, 2015 at 2:32 PM, Sebastian Florek <notifications@github.com>
wrote:

> In src/app/frontend/replicasetdetail/replicasetdetail.html
> <https://github.com/kubernetes/dashboard/pull/127#discussion_r46681410>:
>
> > -    </table>
> > +<md-content layout="row" layout-fill flex>
> > +  <div class="kd-replicasetdetail-sidebar" layout="column">
> > +    <div flex layout-align="start center" layout="row" class="kd-header-option-panel">
> > +      <a flex-offset="10" ui-sref="replicasets">
> > +        <i class="material-icons kd-align-mid kd-font-lg">keyboard_backspace</i>
> > +      </a>
> > +      <span flex-offset="5" flex class="kd-align-mid md-title">
> > +        {{ctrl.replicaSetDetail.name}}
> > +      </span>
> > +    </div>
> > +    <div flex-offset="5" flex layout-align="start start" layout-padding layout="row">
> > +      <div flex layout="column">
> > +        <h3 layout="row" class="kd-primary md-body-2">
> > +          <div flex="55">
> > +            <i class="kd-align-mid material-icons">refresh</i>
>
> All the icon examples on material icons are within <i> tag. Is there a
> difference if we use span instead of i for icons?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/dashboard/pull/127/files#r46681410>.
>